taker-frontend/src/components/Trade.tsx

```diff
@@ -221,8 +222,20 @@ export default function Trade({
                                     </Td>
                                 </Tr>
                                 <Tr>
-                                    <Td>Perpetual Cost</Td>
-                                    <Td isNumeric>{fundingRateHourly ? fundingRateHourly + "%" : "not available"}</Td>
+                                    <Td><Text>Perpetual Cost</Text></Td>
+                                    <Tooltip
+                                        label={`The CFD is rolled over perpetually every hour at ${fundingRateHourly}%, annualized that is ${fundingRateAnnualized}%. 
+                                        The funding rate can fluctuate depending on the market movements.`}
```

Comment:
```suggestion
These costs fluctuate depending on the market movements.
```

Otherwise this is the first mention of "funding rate".